### DISCUSSION TITLE
It appears you are not eligible to sign up for GitHub Copilot Free

### DISCUSSION THREAD

**Discussion by ljy-life** [community/community#147478] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

<img width="671" alt="image" src="https://github.com/user-attachments/assets/79de86e4-1f55-4ac1-93eb-cb133c08649f" />

my  email  1535664756@qq.com 
And the copilot is for free,but i can't use it. I don't know why?
please help me. 3ku

**Comment by milekv** [comment]
Try download VSCode and install Copilot on this, also u can use Copilot here "https://github.com/copilot"

**Reply by ljy-life** [reply]
yes  itry it, but you konw,It doesn't work.

**Reply by milekv** [reply]
So "learn more" ? what u see here? maybe some wrong region that github copilot doesn't support

**Reply by ljy-life** [reply]
it's  pay for copilot pro website, I'm a personal.

![image](https://github.com/user-attachments/assets/1f99cb27-397d-43ab-8abe-d259290a98a0)

**Reply by milekv** [reply]
everything seems correct but you still can't access Copilot Free, imo u should contact [GitHub Support](https://support.github.com/). Explain your situation clearly and include screenshots (like this one) to help them diagnose the problem. **_((maybe try to create an opensource project because the maintainers are definitely eligible, or try to create a new account in incognito mode and go to the github copilot free website there))_**

**Reply by ljy-life** [reply]
I just want to use this account , I don't want to give up my account. Beacuse all of my other app use this account. 
May be i need to create an new account for free copilot.
Thank you, My man.

**Reply by milekv** [reply, marked as answer]
I know, but you can have a second account and have the repositories from this account downloaded to a laptop with VSCod running on the second account and then upload them back to the original one

**Reply by milekv** [reply]
I will be grateful if you click "mark as answer" and we will close the chat this way

**Reply by ljy-life** [reply]
The question is  stuident free account, I see the same question in the other discussions. 
Thank you.

**Reply by spraylee** [reply]
i have the same problom, is your problem solved？ @ljy-life

**Reply by ljy-life** [reply]
sorry, no

**Comment by jason-lee-x** [comment]
same here.Could anyone tell me why?

![image](https://github.com/user-attachments/assets/f39b9dd8-31c6-4a23-8ec1-88e692923717)

**Reply by ljy-life** [reply]
May be we used to a student?
